spec/amber/validations/params_spec.cr

```diff
@@ -12,8 +12,40 @@ module Amber
           required("last_name") { |v| v.str? & !v.empty? }
         end
 
-        validator.errors.size.should eq 0
         validator.valid?.should be_true
+        validator.errors.size.should eq 0
+      end
+
+      context "optional params" do
+        context "when missing" do
+          it "does not validate optional field" do
```

Comment:
As a rule, each context should have at least 2 tests; one positive and one negative.  Can you add a test for when `validator.valid?.should be_true`?